src/client/app/shared/card/card-renderer.component.ts

```diff
@@ -0,0 +1,59 @@
+import { Component, Input, ElementRef } from '@angular/core';
+import './Card';
+
+/**
+ * This class represents the CardRenderer component.
+ */
+@Component({
+  moduleId: module.id,
+  selector: '[card-renderer]',
+  templateUrl: 'card-renderer.component.html',
+  styleUrls: ['card-renderer.component.css']
+})
+export class CardRendererComponent {
```

Comment:
I take this back from before: we should just call this class simply "card.component.ts".

"Renderer" can be implied from the fact that it's a component.